second-edition/src/ch10-03-lifetime-syntax.md

```diff
@@ -200,38 +200,38 @@ fn longest(x: &str, y: &str) -> &str {
 }
 ```
 
-<span class="caption">Listing 10-22: An implementation of the `longest`
-function that returns the longest of two string slices, but does not yet
-compile</span>
+<span class="caption">Listado 10-22: una implementación de la función más
+`larga` que devuelve lo más largo de dos segmentos de cadena, pero aún no
+se compila</span>
 
-Instead we get the following error that talks about lifetimes:
+En cambio, obtenemos el siguiente error que habla de tiempos de vida:
 
 ```text
-error[E0106]: missing lifetime specifier
+error[E0106]: falta especificador de vida 
    |
 1  | fn longest(x: &str, y: &str) -> &str {
-   |                                 ^ expected lifetime parameter
+   |                                 ^ parámetro de vida esperado
    |
-   = help: this function's return type contains a borrowed value, but the
-   signature does not say whether it is borrowed from `x` or `y`
+   = ayuda: el tipo de devolución de esta función contiene un valor prestado, pero la
```

Comment:
Not sure if this is needed to translate in this case, you can leave the original